challenges/objects-arrays.js

```diff
@@ -59,11 +83,17 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name + ' ' + graduates[i].email);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i < graduates.length; i++) {
+  uni.slice(graduates[i].university);
+}
```

Comment:
`Find out how many universities have the string "Uni" included in their name.` ;)
F/e the string `'International Medical & Technological University' `  does have 'Uni' in  it.